howto/anbox/develop-addon.md

```diff
@@ -0,0 +1,46 @@
+Developing and testing addons using the Anbox Management Service (AMS) may be time-consuming. Instead, a container with `--devmode` enabled can provide a safe environment to test addons and their scripts without having to upload the addon to the AMS.
+
+This guide explains how to use a container in development mode to develop and test an addon using Anbox runtime. See [development mode](https://discourse.ubuntu.com/t/17763#dev-mode) to learn more about development mode enabled containers.
+
+## Launch a container in development mode
+
+Start a raw container with `--devmode` enabled:
+
+```
+amc launch --devmode --raw
+```
+
+The command prints out the ID of the container. Note down the container ID for next steps.
+
+Use the `amc shell <container_id>` command to open a shell inside the container. `container_id` is the container ID from the previous step.
+
+Alternatively, you can use `amc exec <container_id> <command_options>` to directly execute commands inside the container.
+
+## Create an addon within the container
+
+Use the container as a remote environment to develop your addon. To make your addon source available within the container, either copy the addon manifest and hooks using the [`lxc file push`](https://documentation.ubuntu.com/lxd/en/latest/howto/instances_access_files/#push-files-from-the-local-machine-to-the-instance) command or clone a git repository using SSH.
+
+You can test your addon scripts by running it inside the container shell. For example, `ADDON_DIR=$PWD ./hooks/install` can help test if the install hook of the addon works.
```

Comment:
As `ADDON_DIR` is mentioned here, probably it's worth mentioning all available environment vars which can be used in the hook here https://anbox-cloud.io/docs/ref/hooks#env-variables